arlo-client/src/components/CreateAudit.tsx

```diff
@@ -41,6 +54,35 @@ const CreateAudit = ({ history }: RouteComponentProps<ICreateAuditParams>) => {
       toast.error(err.message)
     }
   }
+
+  const [elections, setElections] = useState<IElections>({ elections: [] })
+
+  const getStatus = useCallback(async (): Promise<IElections> => {
+    try {
+      const list: IElections | IErrorResponse = await api(`/elections`)
+      if (checkAndToast(list)) {
+        return { elections: [] }
+      } else {
+        return list
+      }
+    } catch (err) {
+      toast.error(err.message)
+    }
+    return { elections: [] }
+  }, [])
+
+  const updateElections = useCallback(async () => {
+    const list = await getStatus()
+    setLoading(true)
+    setElections(list)
+    setElectionId(list.elections.length ? list.elections[0].id : '')
+    setLoading(false)
```

Comment:
This `setLoading` call will negate the `setLoading` call above, right? I'm reasonably sure React will not re-render for each of these state changes but will coalesce them.